src/UI/Pages/Agents/AgentsTablePresenter.ts

```diff
@@ -0,0 +1,77 @@
+import { Agent, AgentRow } from "@/Core";
+import { ColumnHead, TablePresenter } from "@/UI/Presenters";
+import { words } from "@/UI/words";
+
+export class AgentsTablePresenter implements TablePresenter<Agent, AgentRow> {
+  readonly columnHeads: ColumnHead[];
+  readonly numberOfColumns: number;
+
+  constructor() {
+    this.columnHeads = [
+      { displayName: words("agents.columns.name"), apiName: "name" },
+      { displayName: words("agents.columns.process"), apiName: "process_name" },
+      { displayName: words("agents.columns.status"), apiName: "status" },
+      {
+        displayName: words("agents.columns.failover"),
+        apiName: "last_failover",
+      },
+      {
+        displayName: words("agents.columns.unpause"),
+        apiName: "unpause_on_resume",
+      },
+      { displayName: words("agents.columns.actions"), apiName: "actions" },
+    ];
+    this.numberOfColumns = this.columnHeads.length + 1;
+  }
+
+  createRows(sourceData: Agent[]): AgentRow[] {
+    return sourceData.map(
```

Comment:
This map function doesn't do anything except for filter out the environment name.
This can be made more readable by using a lodash method to just drop the environment name.